device/config/constants.go

```diff
@@ -0,0 +1,61 @@
+package config
+
+const (
+	// Default Configurator constants that are describe a specific configuration option
```

Comment:
__[golint]__
* [golint] comment on exported const Locale should be of the form "Locale ..."

<!-- Comment by [SideCI](https://sideci.com) -->